pkgs/development/libraries/libusb1/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchurl, pkgconfig, systemd ? null, libobjc, IOKit }:
 
 stdenv.mkDerivation rec {
-  name = "libusb-1.0.22";
+  name = "libusb-${version}";
+  version = "1.0.22";
 
   src = fetchurl {
-    url = "mirror://sourceforge/libusb/${name}.tar.bz2";
+    url = "https://github.com/libusb/libusb/releases/download/v${version}/${name}.tar.bz2";
```

Comment:
Couldn't fetchFromGitHub here be used instead? The release pages don't seem to contain anything other than just the source code.

Reply:
Actually, there seems to be some difference in the source archive and release tarball: https://gist.github.com/lschuermann/61ab11d3cc7ae3f9073f0c0e1da04981.
I think most of these files could be generated using autoreconfHook, however I'm not quite sure whether a release tarball or building from the source archive is preferred.

This is the same situation as in #48381.

Reply:
I'd prefer to use the source tarballs (non-release). This way the build will also succeed if somebody overrides src with something else.

Reply:
Good point. I'll change it to use `fetchFromGitHub`.